### PR TITLE
Adds MobX / MobX React Lite

### PR DESCRIPTION
**Pull request by ericmasiello**
@dai-shi I could not generate the JSON output when running all the tests together.  Thus, I was unable to update the README with MobX. If I comment out all the other test cases and only run `'mobx'` it works as expected. Please advise if you have any guidance on how to update the README for everything.

Thank you

### CODE REVIEW

**Comment by dai-shi** [comment]
Can you please check this? https://github.com/dai-shi/will-this-react-global-state-work-in-concurrent-rendering/pull/63#issuecomment-1051990738

**Comment by ericmasiello** [comment]
> Can you please check this? [#63 (comment)](https://github.com/dai-shi/will-this-react-global-state-work-in-concurrent-rendering/pull/63#issuecomment-1051990738)

Fixed!

File: __tests__/all_spec.js [hunk 1]
```diff
@@ -15,6 +15,7 @@ jest.setTimeout(20 * 1000);
 
 const names = [
   // 'react-state',
+  'mobx',
```
Comment (dai-shi) [inline __tests__/all_spec.js:18]: Please add it to the end of the array.

**Review by dai-shi — COMMENTED** [review]
> I could not generate the JSON output when running all the tests together.

I don't know... any error message? Does it work without `mobx`?

File: __tests__/all_spec.js (hunk 1 above)
Comment (ericmasiello) [reply __tests__/all_spec.js]: Fixed!

**Comment by ericmasiello** [comment]
> > I could not generate the JSON output when running all the tests together.
> 
> I don't know... any error message? Does it work without `mobx`?

Disregard. It works OK now.

**Review by dai-shi — APPROVED** [review]
LGTM. Thank you!

**Pull request merged**